app/workers/email_held_bike_notification_worker.rb

```diff
@@ -0,0 +1,27 @@
+class EmailHeldBikeNotificationWorker < ScheduledWorker
+  def self.frequency
+    24.hours
+  end
+
+  def perform
+    record_scheduler_started
+    notify_of_bike_index_held
+    record_scheduler_finished
+  end
+
+  def notify_of_bike_index_held
+    Bike.held_with_match.each do |bike, match|
+      next if CustomerContact.held_bike_notification.exists?(bike_id: bike.id, user_email: bike.owner_email)
```

Comment:
I believe there should be additional data stored (maybe in customer contact's info_hash):

- I think it's reasonable to use the `held_bike_notification`s to track the held matches, we need to record enough things to differentiate different matches. e.g. a bike is recovered then stolen again, a match is found but turns out to be incorrect and another match is found. So we at least need to store the `stolen_record_id`, the `match_id` and the `match_source` (whether it's Bike Index or somewhere else)
- `CustomerContact` record should be sufficient to send the email (and should be the thing that is passed in to the mailer)